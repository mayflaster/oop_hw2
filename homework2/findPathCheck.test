CreateNode n1 5
CreateNode n2 10
CreateNode n3 1
CreateGraph A
AddNode A n1
AddNode A n2 
AddNode A n3
AddEdge A n3 n1 
AddEdge A n1 n3
AddEdge A n1 n2
ListNodes A
ListChildren A n1
AddEdge A n3 n3
ListChildren A n3
FindPath A n3 -> n2

AddEdge A n3 n2
FindPath A n3 -> n2

FindPath A n2 -> n1
FindPath A n3 n1 -> n1 n2

CreateNode n4 40
FindPath A n4 -> n1
FindPath A n1 -> n4

FindPath A n3 n1 -> n3 n2