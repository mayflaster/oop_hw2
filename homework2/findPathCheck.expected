created node n1 with cost 5
created node n2 with cost 10
created node n3 with cost 1
created graph A
added node n1 to A
added node n2 to A
added node n3 to A
added edge from n3 to n1 in A
added edge from n1 to n3 in A
added edge from n1 to n2 in A
A contains: n1 n2 n3
the children of n1 in A are: n2 n3
added edge from n3 to n3 in A
the children of n3 in A are: n1 n3
shortest path in A: n3 n1 n2

added edge from n3 to n2 in A
shortest path in A: n3 n2

no path found in A
shortest path in A: n1

created node n4 with cost 40
no path found in A
no path found in A

shortest path in A: n3
 